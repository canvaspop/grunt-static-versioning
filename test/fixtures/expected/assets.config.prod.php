<?php
return array(
    'static.assets' => array(
        'global' => array(
            'css' => array(
                '/css/main.b6f17edb.css',
            ),
            'js' => array(
                '/js/main.cce1a4ed.js',
                '/js/plugin.71f3b3f2.js',
            ),
        ),
        'all' => array(
            'css' => array(
            ),
            'js' => array(
                '/js/all.3039dd25.js',
            ),
        ),
    )
);
