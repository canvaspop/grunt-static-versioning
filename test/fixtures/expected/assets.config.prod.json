{
  "staticAssets": {
    "global": {
      "js": [
        "/main.cce1a4ed.js",
        "/plugin.71f3b3f2.js",
        "/skip.js"
      ],
      "css": [
        "/main.b6f17edb.css"
      ]
    },
    "all": {
      "js": [
        "/all.3039dd25.js"
      ],
      "css": []
    }
  }
}